monai/apps/reconstruction/complex_utils.py

```diff
@@ -121,6 +121,21 @@ def complex_abs(x: NdarrayOrTensor) -> NdarrayOrTensor:
     return (x[..., 0] ** 2 + x[..., 1] ** 2) ** 0.5
 
 
+def complex_abs_t(x: Tensor) -> Tensor:
```

Comment:
it's the same as `complex_abs`, let's not repeat the logic, perhaps we can remove the type annotations for now.